src/store/modules/backend.js

```diff
@@ -15,6 +15,10 @@ export default {
     tip: {},
     comment: {},
     stats: null,
+    prices: {},
+  },
+  getters: {
+    minTipAmount: ({ prices: { usd } }) => 0.01 / usd || 0.06,
```

Comment:
What is this `0.06` for?

The requirement was to be able to tip `0.01` USD or more 🤔

Reply:
This value will be used until rates are fetched. I have just calculated it by the current rates. I think it should work fine even without a default value, should I remove that? Previously the default value was `0.01` that is also no enough clear.

Reply:
But the prices are already being fetched

Reply:
I have removed that and rebased in at the top of develop.